### PR TITLE
crio: Make oci umount configurable

### PR DESCRIPTION
**Pull request by mrunalp**
@ashcrow @giuseppe @runcom @mwoodson @sdodson PTAL

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by runcom** [comment]
:+1:

**Comment by ashcrow** [comment]
Unrelated

```
        "tar: /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v3.10/: Cannot open: No such file or directory", 
        "tar: Error is not recoverable: exiting now"
```

and

```
    "msg": "last_checked_host: localhost, last_checked_var: openshift_image_tag;openshift_image_tag must be in the format\nv#.#.#[-optional.#]. Examples: v1.2.3, v3.5.1-alpha.1\nYou specified openshift_image_tag=valpha.4"
```

File: roles/container_runtime/tasks/package_crio.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@
     dest: /etc/sysconfig/crio-network
     src: crio-network.j2
 
+- name: Configure oci umount for CRI-O
+  lineinfile:
+    dest: /etc/oci-umount.conf
+    line: '{{ item }}'
+  with_items: "{{ container_runtime_oci_umounts }}"
```
Comment (sdodson) [inline roles/container_runtime/tasks/package_crio.yml]: Don't we need a default empty list here to avoid undefined errors?

Comment (sdodson) [reply roles/container_runtime/tasks/package_crio.yml]: `with_items: "{{ container_runtime_oci_umounts | default({}) }}"`

**Pull request closed without merging**